app/__init__.py

```diff
@@ -25,7 +27,7 @@ def null_to_dash(value):
     return value
 
 
-def create_app(config_class):
+def create_app(config_class, database_uri=None):
```

Comment:
@colinbowen we know this seems a bit hacky but it was the easiest way to be able to manually set the db url in the Test App crated by our app fixture since we spin up a test postgres db using testing.postgres, and so the url is dynamic and cant be defined in the Config before runtime.

Reply:
Sure, something to just be aware of so thanks for the heads up